quic/s2n-quic-platform/src/message/msg/tests.rs

```diff
@@ -34,6 +34,22 @@ fn test_msghdr<F: FnOnce(&mut msghdr)>(f: F) {
     f(&mut msghdr);
 }
 
+#[cfg(kani)]
+#[allow(dead_code)] // Avoid warning when using stubs.
+mod stubs {
+    use s2n_quic_core::inet::AncillaryData;
+
+    pub fn decode(_msghdr: &libc::msghdr) -> AncillaryData {
+        let mut ancillary_data = AncillaryData::default();
+        ancillary_data.ecn = kani::any();
+        ancillary_data.local_address = kani::any();
+        ancillary_data.local_interface = Some(kani::any());
```

Comment:
We should probably alternate on the option, yes?
```suggestion
        ancillary_data.local_interface = kani::any();
```

Reply:
Implementing `Arbitrary` for `AncillaryData` took care of that.